<ui version="4.0" >
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Package Manager Settings</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../data/data.qrc" >
    <normaloff>:/data/package-manager.png</normaloff>:/data/package-manager.png</iconset>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1" >
      <attribute name="title" >
       <string>&amp;General Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <widget class="QCheckBox" name="onlyGuiApp" >
         <property name="text" >
          <string>&amp;Show only desktop applications</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QCheckBox" name="systemTray" >
         <property name="text" >
          <string>&amp;Enable system tray icon</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QGroupBox" name="updateCheckGroupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Update check</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" colspan="3" >
           <widget class="QCheckBox" name="intervalCheck" >
            <property name="text" >
             <string>Enable &amp;periodic update check</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="intervalLabel" >
            <property name="text" >
             <string>Check interval:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="intervalSpin" >
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix" >
             <string> min</string>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>10000</number>
            </property>
            <property name="value" >
             <number>15</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>305</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1" >
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QGroupBox" name="bandwidthLimitGroupBox" >
         <property name="title" >
          <string>Bandwidth limit</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <item row="0" column="0" colspan="3" >
           <widget class="QCheckBox" name="useBandwidthLimit" >
            <property name="text" >
             <string>Enable &amp;periodic update check</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="bandwidthLabel" >
            <property name="text" >
             <string>Check interval:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="bandwidthSpin" >
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix" >
             <string> kb/s</string>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>20000</number>
            </property>
            <property name="value" >
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <spacer name="horizontalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>304</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1" >
           <spacer name="verticalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>C&amp;ache</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="updateCheckGroupBox_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Update check</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_5" >
          <item row="0" column="0" colspan="4" >
           <widget class="QCheckBox" name="useCacheCheck" >
            <property name="text" >
             <string>&amp;Use hard disk cache for downloaded software</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="useCacheLabel" >
            <property name="text" >
             <string>Maximum cache size:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="useCacheSpin" >
            <property name="minimumSize" >
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix" >
             <string> Mb</string>
            </property>
            <property name="minimum" >
             <number>0</number>
            </property>
            <property name="maximum" >
             <number>10000</number>
            </property>
            <property name="value" >
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLabel" name="useCacheLimitTip" >
            <property name="text" >
             <string>(use 0 for no limit)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>330</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1" >
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="clearCacheGroupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Clear Cache</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_6" >
          <item row="0" column="0" colspan="2" >
           <widget class="QLabel" name="clearCacheDesc" >
            <property name="text" >
             <string>Clearing the disk cache will remove all downloaded software. 
This will not delete installed software or affect your system stability.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QPushButton" name="clearCacheButton" >
            <property name="text" >
             <string>C&amp;lean disk cache now</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>330</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" >
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>&amp;Repositories</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>&amp;Proxy</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="buttonHelp" >
       <property name="text" >
        <string>&amp;Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>onlyGuiApp</tabstop>
  <tabstop>systemTray</tabstop>
  <tabstop>intervalCheck</tabstop>
  <tabstop>intervalSpin</tabstop>
  <tabstop>useBandwidthLimit</tabstop>
  <tabstop>bandwidthSpin</tabstop>
 </tabstops>
 <resources>
  <include location="../data/data.qrc" />
 </resources>
 <connections/>
</ui>
